common-util/src/main/java/org/sunbird/common/models/util/JsonKey.java

```diff
@@ -705,6 +712,7 @@
   public static final String PRIVATE_FIELDS = "privateFields";
   public static final String SUNBIRD_USER_PROFILE_FIELD_DEFAULT_VISIBILITY =
       "sunbird_user_profile_field_default_visibility";
+  public static final String SUNBIRD_STORE_API_BASE_URL = "sunbird_store_api_base_url";
```

Comment:
Why a new variable is required? Can't existing env SUNBIRD_API_BASE_URL be used?